creator-node/test/ffmpeg.js

```diff
@@ -0,0 +1,95 @@
+const { segmentFile } = require('../src/ffmpeg')
+
+const path = require('path')
+const fs = require('fs')
+const assert = require('assert')
+
+describe('test segmentFile', () => {
+  // Create the segments directory to store segments in
```

Comment:
i'd also. just explain why you do this here. this is because the middleware usually does this step but if you call it without the middleware it needs to be done by hand